src/java/org/apache/cassandra/index/sai/iterators/KeyRangeIterator.java

```diff
@@ -43,6 +43,11 @@ protected KeyRangeIterator(Builder.Statistics statistics)
         this(statistics.min, statistics.max, statistics.count);
     }
 
+    public KeyRangeIterator(KeyRangeIterator range)
+    {
+        this(range == null ? null : range.min, range == null ? null : range.max, range == null ? -1 : range.count);
```

Comment:
Nit: can we break the args for better readability?
```suggestion
        this(range == null ? null : range.min,
             range == null ? null : range.max,
             range == null ? -1 : range.count);
```